web/packages/demo/www/index.html

```diff
@@ -15,7 +15,7 @@
                         src="https://ruffle.rs/assets/logo.png"
                         alt="Ruffle"
                         data-canonical-src="https://ruffle.rs/assets/logo.png"
-                    />
+                        height="32" />
```

Comment:
Is this more appropriate in CSS?

Reply:
Honestly I don't have a strong preference. I'll go with your decision.

Reply:
Yea, I'd say let's toss this in the CSS.